README.md

```diff
@@ -2,6 +2,24 @@
 
 # Terraform Academy
 
-
 # Template Examples
+
 https://github.com/gitpitch/the-template/blob/master/PITCHME.md
+
+## Using Cloud9
+
+Use the following CloudFormation template to setup a Cloud9 environment in us-east-1 (N. Virginia), just click _Next_, _Next_, _Next_ and _Create Stack_:
+
+[![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=TerraformAcademy&templateURL=https://cloudtitlan-public-cfn-templates.s3.amazonaws.com/terraform-academy.yaml)
```

Comment:
please change this to use a bucket in the wize-services AWS account

Reply:
done!